src/message_replies/reportChannelPoints.js

```diff
@@ -70,4 +70,64 @@ const reportChannelPoints = async (interaction) => {
   }
 };
 
-export default reportChannelPoints;
+const reportGlobalPoints = async (interaction) => {
```

Comment:
It all looks good to me. Any interest in refactoring this to one method that can handle global or channel? Or will your next PR make this comment irrelevant?

Reply:
well, the new mode is based on filters, for the global points you would need only to pass 2 different filter, but in the end of the day, I think that having 1 or 2 functions in this case will probably be the same. if we got to a point where we're using more than 2 filters, which I can't see how, we can create a more generic function.